test/fixtures/render/enum-sample.json

```diff
@@ -204,7 +204,7 @@
                                   "content": "application/json"
                                 }
                               },
-                              "content": "{\n  \"enumField\": \"c\"\n}"
+                              "content": "{\n  \"enumField\": \"a\"\n}"
```

Comment:
> I think this is a regression. Before we was using the default value `c` and now we're using the first enumeration value. I believe the default should take precedence.

Similar to above, I think the sample value should take precedence over the enumeration members.